benchmarks/fastrnns/bench.py

```diff
@@ -207,6 +207,8 @@ def bench_group(model_list, bench_name, bench_group, bench_args):
 
     args = parser.parse_args()
     assert args.fuser in ['te', 'old', 'none']
+    if args.fuser == 'te':
+        torch._C._jit_register_tensorexpr_fuser()
```

Comment:
Now think about this. I think "_jit_set_texpr_fuser_enabled" should imply "_jit_register_tensorexpr_fuser"

Reply:
Hmm, that's actually a good idea.  We don't need both, and set_enabled is more useful since you might want to toggle it on and off.  I'll do that instead of this.